docs/sdk/src/reference_docs/omni_node.rs

```diff
@@ -0,0 +1,170 @@
+//! # (Omni) Node
+//!
+//! This reference docs elaborates on what a Polkadot-SDK/Substrate node software is, and what
+//! various ways exist to run one.
+//!
+//! The node software, as denoted in [`crate::reference_docs::wasm_meta_protocol`], is everything in
+//! a blockchain other than the WASM runtime. It contains common components such as the database,
+//! networking, RPC server and consensus. Substrate-based nodes are native binaries that are
+//! compiled down from the Rust source code. The `node` folder in any of the [`templates`] are
+//! examples of this source.
+//!
+//! <details>
+//! <summary>
+//! A typical node also contains a lot of other tools (exposed as subcommands) that are useful for
+//! operating a blockchain:
+//! </summary>
+// #![doc = docify::embed!("../../substrate/bin/node/cli/src/cli.rs", "Subcommand")]
+//!
+//! </details>
+//!
+//! ## Node <> Runtime Interdependence
+//!
+//! While in principle the node can be mostly independent of the runtime, for various reasons, such
+//! as the [native runtime](crate::reference_docs::wasm_meta_protocol#native-runtime), the node and
+//! runtime were historically tightly linked together. Another reason is that the node and the
+//! runtime need to be in agreement about which consensus algorithm they use, as described
+//! [below](#consensus-engine).
+//!
+//! Specifically, the node relied on the existence of a linked runtime, and *could only reliably run
+//! that runtime*. This is why if you look at any of the [`templates`], they are all composed of a
+//! node, and a runtime.
+//!
+//! Moreover, the code and API of each of these nodes was historically very advanced, and tailored
+//! towards those who wish to customize many of the node components at depth.
+//!
+//! <details>
+//! <summary>
+//! The notorious <pre>service.rs</pre> is a good example of this:
+//! </summary>
+// #![doc = docify::embed!("../../substrate/bin/node/src/cli/service.rs", "new_full_base")]
+//!
+//! </details>
+//!
+//! A [trend](https://github.com/paritytech/polkadot-sdk/issues/62) has already been undergoing in
+//! order to de-couple the node and the runtime for a long time. The north star of this effort is
+//! twofold :
+//!
+//! 1. develop what can be described as an "omni-node": A node that can run all runtimes.
+//! 2. provide a cleaner abstraction for creating a custom node.
+//!
+//! While a single omni-node running *all possible runtimes* is not feasible, the
+//! [`polkadot-omni-node`] is an attempt at creating the former, and the [`polkadot-omni-node-lib`]
+//! is the latter.
+//!
+//! > The OmniNodes are mainly focused on the development needs of Polkadot parachains, not
```

Comment:
This looks strange when rendedred? Is it intentionally indented?
I would also highlight somehow information that omni-node supoorts parachains only (maybe warning emoji or some frame (don't know if rust doc allows for doc)).